packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -204,22 +204,16 @@ export default class MeetingInfoV2 {
       return invitees;
     };
 
-    return this.webex.internal.conversation
-      .get({url: conversationUrl}, {includeParticipants: true, disableTransform: true})
-      .then((conversation) => {
-        const body: {
-          title: string;
-          spaceUrl: string;
-          keyUrl: string;
-          kroUrl: string;
-          invitees: any[];
-          installedOrgID?: string;
-        } = {
+    return this.webex
+      .request({uri: conversationUrl, qs: {includeParticipants: true}, disableTransform: true})
+      .then(({body: conversation}) => {
+        const body = {
           title: conversation.displayName,
           spaceUrl: conversation.url,
           keyUrl: conversation.encryptionKeyUrl,
           kroUrl: conversation.kmsResourceObjectUrl,
           invitees: getInvitees(conversation.participants?.items),
+          installedOrgID,
```

Comment:
adding `installedOrgID` here means that lines 219-221 can be removed, but it means a slight change in behavior, for example if installedOrgID was an empty string, before this PR it wouldn't be set in the body and now it will be. @Coread do you know if that's OK?

Reply:
installedOrgID will be checked for policies. Presumably an orgID of blank will not match any policies and so will not do anything. Not sure the reason for this change though